docs/blog/2019-05-07-advanced-sitemap-plugin-for-seo/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Advanced Sitemap Plugin for SEO
+date: 2019-05-07
+author: Aileen Nowak
+excerpt: "Ghost migrated their docs site to Gatsby and built an automatic XML sitemap plugin which is now open source and available for anyone to use!"
+tags:
+  - ghost
+  - seo
+  - sitemap
+  - headless cms
+---
+
+In recent months, I’ve been working on projects for [Ghost](https://ghost.org/) with a JAMstack approach and using the Gatsby framework to build static sites. It’s been an exciting journey that began when we rebuilt our [docs](https://docs.ghost.org/) site from the ground up – now we’re migrating all of our sites to use the same technology.
+
+It’s been a pleasure working with this stack and getting to grips with a static site generator like Gatsby. At Ghost, we have a small team of engineers, designers and marketers who collaborate on projects like this to ensure the end result hits the mark.
+
+One of the things I was working on was implementing performance enhancements and SEO best practice to ensure our new docs site ranked well in the search engines. This included pulling in custom meta data and structured data via the Ghost API (which we are using as a Headless CMS), and implementing XML sitemaps.
+
+I wanted to recreate the default advanced sitemap structure that we use for all Ghost publications on Gatsby, and since it’s an open source project, I was able to go ahead and build a [plugin](https://www.gatsbyjs.org/packages/gatsby-plugin-advanced-sitemap/?=sitemap) which is now available for anyone to use!
```

Comment:
```suggestion
I wanted to recreate the default advanced sitemap structure that we use for all Ghost publications on Gatsby, and since it’s an open source project, I was able to go ahead and build a [plugin](/packages/gatsby-plugin-advanced-sitemap/?=sitemap) which is now available for anyone to use!
```